tasklist/qa/integration-tests/src/test/java/io/camunda/tasklist/graphql/ProcessSaasIT.java

```diff
@@ -70,7 +70,7 @@ public void shouldReturnAllProcessesBasedOnQueries() throws IOException {
         responseProcessDefinition.get("$.data.processes[0].processDefinitionId"));
 
     LOGGER.info("Should return all process based on Process id query");
-    testProcessRetrieval("2251799813685250", 1);
+    testProcessRetrieval("2251799813685251", 1);
```

Comment:
The expected process definition key changed in the process engine, i.e., it generates another key than expected.